server/src/routes/api/v1/showsReviewsRouter.js

```diff
@@ -24,4 +28,33 @@ reviewsRouter.post("/", async (req,res) => {
   }  
 })
 
+reviewsRouter.delete("/:id", async (req, res) => {
+  try {
+    const reviewId = req.params.id
+    await Review.query().deleteById(reviewId)
+    return res.status(200).json()
+  } catch(error) {
+    return res.status(500).json({ error })
+  }
+})
```

Comment:
this ebdpoint can be in a regular reviewsRouter once you update the pathing in your delete Fetch request

Reply:
we currently do not have a reviewsRouter, whereas the other group does. we aim to move this endpoint into the reviewsRouter after we've merged to main and can see what is already present in reviewsRouter.